client/state/data-layer/wpcom/domains/validation-schemas/index.js

```diff
@@ -0,0 +1,69 @@
+/** @format */
```

Comment:
[From the docs](https://github.com/Automattic/wp-calypso/tree/master/client/state/data-layer#file-structure)  I understand that file structure in the data-layer should reflect the wpcom api path. Something like `/domains/validation-schema/${ tld }` should be represented as `client/state/data-layer/wpcom/domains/validation-schema/index.js`.